src/amo/css/OverallRating.scss

```diff
@@ -1,39 +1,31 @@
-.OverallRating {
-  form {
+.OverallRating form fieldset {
+  .OverallRating-selectors {
```

Comment:
Looks like this could be `.OverallRating-selectors` instead of `.OverallRating form fieldset .OverallRating-selectors`.